pubsub/google/cloud/pubsub/_http.py

```diff
@@ -203,9 +204,10 @@ def topic_publish(self, topic_path, messages):
         :rtype: list of string
         :returns: list of opaque IDs for published messages.
         """
-        _transform_messages_base64(messages, _base64_unicode)
+        messages_to_send = copy.deepcopy(messages)
```

Comment:
It's [`Topic.commit()`](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/8001d45c2f6cb3fcf583280874b141c83541c50a/pubsub/google/cloud/pubsub/topic.py#L444)